snowpack/src/commands/dev.ts

```diff
@@ -324,19 +335,23 @@ export async function startServer(
       credentials = await readCredentials(config.root);
     } catch (e) {
       logger.error(
-        `✘ No HTTPS credentials found! Missing Files:  ${colors.bold(
-          'snowpack.crt',
-        )}, ${colors.bold('snowpack.key')}`,
+        `✘ No HTTPS credentials found!`,
       );
-      logger.info(`You can automatically generate credentials for your project via either:
+      logger.info(`You can specify HTTPS credentials via either:
+
+  - Including ${colors.yellow('snowpack.crt')} and ${colors.yellow('snowpack.key')} files in your project folder.
+
+  - Specifying the your HTTPS certificate and private key file using the
```

Comment:
this line looks pretty long, you need to be concious oof line length when outputting to the console so that it doesn't wrap. Maybe:

```
You can specify HTTPS credentials via either:
  - Including "snowpack.crt" and "snowpack.key" files in your root directory.
  - Including credentials in your "devOptions.secure" project config. 
 
    You can automatically generate credentials for your project via either:
```

Happy to reword however you'd like, as long as you keep the column width equally tight.

Reply:
I’d just change the order of the bullet points to better reflect that the config option overrides the presence of the files (and perhaps call it the “project root directory” to differentiate from the system root directory). Otherwise, I’m not overly bothered by the exact property name or whether we pass a path or the actual value. (Although having an inconsistency between different ways of configuring Snowpack does sound odd unless there is a technical reason why they cannot be made consistent. It’s the sort of thing that makes it harder for people who are learning a framework.)